src/mono/wasm/debugger/DebuggerTestSuite/Inspector.cs

```diff
@@ -81,12 +81,20 @@ public Task<JObject> WaitFor(string what)
             {
                 if (tcs.Task.IsCompleted)
                 {
+                    Client.CurrentSessionId = new SessionId(tcs.Task.Result["sessionId"]?.Value<string>());
```

Comment:
I'm not sure I understand this. The client is being switched to the session id based on the last message we received? Why is that?

Reply:
When we receive a pause and will process the pause, we want to run the next commands like get properties, evaluate commands, in the session that we are paused.

Reply:
Maybe it would make sense to keep track of the "main thread"'s sessionId, and the related sessions. And also have a `PausedOnSessionId`, which we set/clear on pause/resume.

For future work, it would be useful to know - do we get contextCreated events per worker, or per managed thread? And what does it mean to evaluate a managed expression on a main vs worker context(sessionId) ? I guess, we'll find out more as we try this out more.